com.github.cassidyjames.clairvoyant.json

```
@@ -1,7 +1,7 @@
 {
   "app-id": "com.github.cassidyjames.clairvoyant",
   "runtime": "org.gnome.Platform",
-  "runtime-version": "45",
+  "runtime-version": "master",
```

```suggestion
  "runtime-version": "46",
```

Maybe I should have a dev/master version of the manifest for CI that includes GNOME nightly SDKs? Not sure. But I definitely want to be able to test against the latest-released SDK instead of just master.